scanners/boostsecurityio/baseline/module.yaml

```diff
@@ -0,0 +1,12 @@
+api_version: 1.0
+
+
+id: boostsecurityio/scanner
+name: BoostSecurity Scanner
+namespace: boostsecurityio/scanner
```

Comment:
Based on my notes, i think we settled for

```
id: boostsecurityio/baseline
...
namespace: boostsecurityio/baseline
```

The namespace is not really important as an analysis of this scanner should never reach rules enrichment because it it will be grouped.   But that might create clashes in rules-management